exporter/opensearchexporter/metadata.yaml

```
@@ -5,5 +5,6 @@ status:
   stability:
     alpha: [traces]
     development: [logs]
+  distributions: [contrib, aws]
```

```suggestion
  distributions: [contrib]
```

This exporter is not included in ADOT at this time.